src/game/spawn/player.rs

```diff
@@ -24,19 +23,36 @@ pub struct SpawnPlayer;
 #[reflect(Component)]
 pub struct Player;
 
-fn spawn_player(_trigger: Trigger<SpawnPlayer>, mut commands: Commands, images: Res<ImageAssets>) {
+fn spawn_player(
+    _trigger: Trigger<SpawnPlayer>,
+    mut commands: Commands,
+    images: Res<ImageAssets>,
+    mut texture_atlas_layouts: ResMut<Assets<TextureAtlasLayout>>,
+) {
+    // A texture atlas is a way to split one image with a grid into multiple sprites.
+    // By attaching it to a [`SpriteBundle`] and providing an index, we can specify which section of the image we want to see.
+    // We will use this to animate our player character. You can learn more about texture atlases in this example:
+    // https://github.com/bevyengine/bevy/blob/latest/examples/2d/texture_atlas.rs
+    let layout = TextureAtlasLayout::from_grid(UVec2::splat(32), 6, 2, Some(UVec2::splat(1)), None);
+    let texture_atlas_layout = texture_atlas_layouts.add(layout);
+    let player_animation = PlayerAnimation::new();
+
     commands.spawn((
         Name::new("Player"),
         Player,
         SpriteBundle {
             texture: images[&ImageAsset::Ducky].clone_weak(),
-            transform: Transform::from_scale(Vec3::splat(0.5)),
+            transform: Transform::from_scale(Vec3::splat(8.0)),
```

Comment:
It was already like this before, but technically we don't want to scale Z. But I don't think it actually does anything in 2D so w/e.

Reply:
done